test/PSReadLine/App.config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
```

Comment:
why are these files being remove?

Reply:
It's reasonable to remove these files at any time - this code is just a simple console application for quick manual testing of PSReadLine.